pipeline/outputs/stackdriver_special_fields.md

```diff
@@ -195,104 +150,44 @@ the logEntry will be:
     "jsonPayload": {
         ...
     }
-    "insertId": "test_id"
-    ...
-}
-```
-
-**2. If the insertId is invalid (not non-empty string).** For example:
-```text
-jsonPayload {
-    "logging.googleapis.com/insertId": 12345
+    "operation": {
+        "first": true,
+        "last": true
+    }
     ...
 }
 ```
-The logging agent will log an error and reject the entry.
-
-## SourceLocation
-SourceLocation field contains additional information about the source code location that produced the log entry. The format.
 
-The JSON representation is as followed:
+**If the field itself is not a map, the plugin will leave this field untouched.** For example:
 ```text
 {
-  "file": string,
-  "line": string,
-  "function": string
-}
-```
-
-### Use Cases
-Set the input log as followed:
-```text
-jsonPayload {
-    "logging.googleapis.com/sourceLocation": {
-        "file": "my_file",
-        "line": 123,
-        "function": "foo()"
-    }
+    "logging.googleapis.com/operation": "some string",
     ...
 }
 ```
 the logEntry will be:
 ```text
 {
     "jsonPayload": {
+        "logging.googleapis.com/operation": "some string",
         ...
     }
-    "sourceLocation": {
-        "file": "my_file",
-        "line": 123,
-        "function": "foo()"
-    }
     ...
 }
 ```
 
-## httpRequest
-HttpRequest field is a common proto for logging HTTP requests.
-
-The JSON representation is as followed:
+**If there are extra subfields, the plugin will add the recognized fields to the corresponding field in the LogEntry, and preserve the extra subfields in jsonPayload.** For example:
```

Comment:
```suggestion
If there are extra subfields, the plugin will add the recognized fields to the corresponding field in the LogEntry, and preserve the extra subfields in jsonPayload. For example:
```

Reply:
Fixed in [40631d3](https://github.com/fluent/fluent-bit-docs/pull/1585/commits/40631d3fc506102f79ef4b5a4364c5a47d7b75a9)